src/session/securechannel.rs

```diff
@@ -70,6 +71,42 @@ const AES_BLOCK_SIZE: usize = 16;
 type Aes128CbcEnc = cbc::Encryptor<Aes128>;
 type Aes128CbcDec = cbc::Decryptor<Aes128>;
 
+/// SCP03 AES Session Keys
+#[derive(Serialize, Deserialize)]
+pub struct SessionKeys {
+    /// Session encryption key (S-ENC)
+    pub enc_key: [u8; KEY_SIZE],
+
+    /// Session Command MAC key (S-MAC)
+    pub mac_key: [u8; KEY_SIZE],
+
+    /// Session Respose MAC key (S-RMAC)
+    pub rmac_key: [u8; KEY_SIZE],
+}
+
+impl Zeroize for SessionKeys {
+    fn zeroize(&mut self) {
+        self.enc_key.zeroize();
+        self.mac_key.zeroize();
+        self.rmac_key.zeroize();
+    }
+}
+
+#[cfg(feature = "yubihsm-auth")]
+impl From<yubikey::hsmauth::SessionKeys> for SessionKeys {
```

Comment:
so this brings a dependency on `yubikey.rs` just because of `SessionKeys`, `Challenge` and `Context`, but maybe this isn't the correct way to do this and it would be easier to rely on the inner slice instead?